client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -312,10 +316,13 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 		if ( ! this.iframePort ) {
 			return;
 		}
-
+		if ( this.props.isSiteUsingFSE ) {
```

Comment:
The following works but we need a way to dynamically set the templateIds. But can't seem to get getEntityRecord working in this context.